app/src/main/kotlin/com/intive/picover/parties/view/AddPartyContent.kt

```diff
@@ -22,29 +22,28 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
 import androidx.compose.ui.unit.dp
 import cafe.adriel.voyager.core.screen.Screen
-import cafe.adriel.voyager.hilt.getScreenModel
 import com.intive.picover.R
 import com.intive.picover.common.text.PicoverOutlinedTextField
 import com.intive.picover.common.validator.TextValidator
 import com.intive.picover.main.navigation.NavControllerHolder
-import com.intive.picover.parties.viewmodel.PartiesViewModel
+import com.intive.picover.parties.model.AddPartyResult
 
 class AddPartyBottomSheet : Screen {
 
 	@OptIn(ExperimentalMaterial3Api::class)
 	@Composable
 	override fun Content() {
-		val viewModel = getScreenModel<PartiesViewModel>()
-		val state by viewModel.state.collectAsState()
+		val (title, setTitle) = remember { mutableStateOf("") }
+		val (description, setDescription) = remember { mutableStateOf("") }
```

Comment:
I think it doesn't make much sense to hoist that in view model if there's no business logic around it

Reply:
Fully agree